XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/Simulation/SimulatedControllerDataProviderProfile.cs

```diff
@@ -131,13 +130,5 @@ public class SimulatedControllerDataProviderProfile : BaseMixedRealityController
         public float RotationSpeed => rotationSpeed;
 
         #endregion
-
-        [SerializeField]
-        private ControllerDataProviderConfiguration[] registeredControllerDataProviders = new ControllerDataProviderConfiguration[0];
-
-        /// <summary>
-        /// The currently registered controller data providers for simulation.
-        /// </summary>
-        public ControllerDataProviderConfiguration[] RegisteredControllerDataProviders => registeredControllerDataProviders;
```

Comment:
These are registered in the `ControllerDataProviderProfile` with the rest of the controllers